src/GPRWrap/GPRWrap.jl

```diff
@@ -0,0 +1,148 @@
+using Parameters # lets you have defaults for fields
+
+using EllipsisNotation # adds '..' to refer to the rest of array
+import ScikitLearn
+import StatsBase
+const sklearn = ScikitLearn
+
+sklearn.@sk_import gaussian_process : GaussianProcessRegressor
+sklearn.@sk_import gaussian_process.kernels : (RBF, Matern, WhiteKernel)
+
+
+@with_kw mutable struct GPRWrap
+  """
+  A simple struct to handle Gaussian Process Regression related stuff
+
+  """
```

Comment:
Julia docstrings go before the function/type

Reply:
whoops, thanks! I'll change that